resources/img/haruna/haruna.mqo

```diff
@@ -1,35 +1,35 @@
-	"�haruna10" col(1 1 1 1) dif(1) amb(1) emi(0) spc(0) power(0) tex("�b��.bmp")
-	"�haruna11" col(1 1 1 1) dif(1) amb(1) emi(0) spc(0) power(0) tex("��s�b��.bmp")
-	"�haruna12" col(1 1 1 1) dif(1) amb(1) emi(0) spc(0) power(0) tex("�Ȗ͗l.tga")
-	"�haruna13" col(1 1 1 0.98) dif(1) amb(1) emi(0) spc(0) power(0) tex("���J.tga")
-	"�haruna14" col(1 1 1 0.98) dif(1) amb(1) emi(0) spc(0) power(0) tex("����.tga")
-	"�haruna15" col(1 1 1 1) dif(1) amb(1) emi(0) spc(0) power(0) tex("�d�T�P.tga")
-	"�haruna16" col(1 1 1 1) dif(1) amb(1) emi(0) spc(0) power(0) tex("�d�T�Q.tga")
-	"�haruna17" col(1 1 1 1) dif(1) amb(1) emi(0) spc(0) power(0) tex("�d�T�R.tga")
-	"�haruna18" col(0.373 0.569 0.584 0.3) dif(1) amb(0.5084966666666666666666666667) emi(0) spc(0.3) power(0)
-	"�haruna19" col(1 1 1 0.75) dif(1) amb(1) emi(0) spc(0) power(0) tex("light.tga")
-	"�haruna20" col(1 1 1 0.98) dif(1) amb(1) emi(0) spc(0) power(0) tex("�g3.tga")
-	"�haruna21" col(1 1 1 0.98) dif(1) amb(1) emi(0) spc(0) power(0) tex("�g4.tga")
-	"�haruna22" col(1 1 1 0.8) dif(1) amb(1) emi(0) spc(0) power(0) tex("�g4.tga")
-	"�haruna23" col(1 1 1 0.6) dif(1) amb(1) emi(0) spc(0) power(0) tex("�g4.tga")
-	"�haruna24" col(1 1 1 0.4) dif(1) amb(1) emi(0) spc(0) power(0) tex("�g4.tga")
-	"�haruna25" col(1 1 1 0.2) dif(1) amb(1) emi(0) spc(0) power(0) tex("�g4.tga")
-	"�haruna26" col(1 1 1 0.05) dif(1) amb(1) emi(0) spc(0) power(0) tex("�g4.tga")
-	"�haruna27" col(1 1 1 1) dif(1) amb(1) emi(0) spc(0) power(0) tex("��3.tga")
+	"haruna1" col(0.2196078 0.2588235 0.2784314 1) dif(1) amb(0.2522875666666666666666666667) emi(0) spc(0) power(0)
+	"haruna2" col(0.1137255 0.1254902 0.1372549 1) dif(1) amb(0.1267974) emi(0) spc(0) power(0)
+	"haruna3" col(0.5019608 0 0 1) dif(1) amb(0.1673202666666666666666666667) emi(0) spc(0) power(0)
+	"haruna4" col(0.7450981 0.6196079 0.1294118 1) dif(1) amb(0.4980393333333333333333333333) emi(0) spc(0.2) power(0)
+	"haruna5" col(0.454902 0.4431373 0.4196078 1) dif(1) amb(0.4392156666666666666666666667) emi(0) spc(0) power(0)
+	"haruna6" col(0 0 0 1) dif(1) amb(0) emi(0) spc(0) power(0)
+	"haruna7" col(1 1 1 1) dif(1) amb(1) emi(0) spc(0.5) power(80)
+	"haruna8" col(0.2196078 0.2588235 0.2784314 1) dif(1) amb(0.2522876) emi(0) spc(0) power(0)
+	"haruna9" col(1 1 1 1) dif(1) amb(1) emi(0) spc(0) power(0) tex("SbA.bmp")
+	"haruna10" col(1 1 1 1) dif(1) amb(1) emi(0) spc(0) power(0) tex("bA.bmp")
+	"haruna11" col(1 1 1 1) dif(1) amb(1) emi(0) spc(0) power(0) tex("osbA.bmp")
+	"haruna12" col(1 1 1 1) dif(1) amb(1) emi(0) spc(0) power(0) tex("EIl.tga")
+	"haruna13" col(1 1 1 0.98) dif(1) amb(1) emi(0) spc(0) power(0) tex("J.tga")
+	"haruna14" col(1 1 1 0.98) dif(1) amb(1) emi(0) spc(0) power(0) tex("C.tga")
+	"haruna15" col(1 1 1 1) dif(1) amb(1) emi(0) spc(0) power(0) tex("dTP.tga")
+	"haruna16" col(1 1 1 1) dif(1) amb(1) emi(0) spc(0) power(0) tex("dTQ.tga")
+	"haruna17" col(1 1 1 1) dif(1) amb(1) emi(0) spc(0) power(0) tex("dTR.tga")

```

Comment:
If these are supposed to be filenames, there are no .tga files in the repo. I wonder if I've misunderstood what a texture is, or if they're just missing.